openBCIBoard.js

```diff
@@ -49,25 +60,19 @@ function OpenBCIFactory() {
             packetsRead:0,
             looseBytes:0
         };
-        self.moneyBuf = new Buffer('$$$');
+        this.moneyBuf = new Buffer('$$$');
+        this.searchingBuf = this.moneyBuf;
+        // Objects
+        this.writer = null;
         // Numbers
-        self.badPackets = 0;
-        self.bytesIn = 0;
+        this.badPackets = 0;
+        this.bytesIn = 0;
+        this.commandsToWrite = 0;
+        this.sampleCount = 0;
         // Strings
-        self.portName = portName;
+        //self.portName = portName;
```

Comment:
yea because i removed port name as an input option here for now because it was not being used at the moment.